cmd/agentctl2/utils/print_log.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"html/template"
+	"os"
+
+	"github.com/ligato/vpp-agent/cmd/agentctl/utils"
+)
+
+type logType struct {
+	Logger string `json:"logger,omitempty"`
+	Level  string `json:"level,omitempty"`
+}
+
+type LogList []logType
+
+func ConvertToLogList(log string) LogList {
+	data := make(LogList, 0)
+	err := json.Unmarshal([]byte(log), &data)
+
+	if nil != err {
+		utils.ExitWithError(utils.ExitError,
+			errors.New("Failed conver string to json - "+err.Error()))
+	}
+
+	return data
+}
+
+func (ll LogList) PrintLogList() (*bytes.Buffer, error) {
+
+	logLevel := createLogTypeTemplate()
+
+	templates := []*template.Template{}
+
+	templates = append(templates, logLevel)
+
+	return ll.textRenderer(templates)
+}
+
+func createLogTypeTemplate() *template.Template {
+	Template, err := template.New("log").Parse(
```

Comment:
If the template is invalid for some reason it's better to sooner. You can use `Must` to achieve this: https://godoc.org/text/template#Must

Also gets parsed only once since it will be in global scope.